### ISSUE TITLE
Way to hide password

### ISSUE THREAD

**Issue by AttilaVM** [bgschiller/postgres_kernel#22]
I want to avoid to write my database user password into a notebook plain text, is it possible?

**Comment by bgschiller** [comment]
Definitely. If you have an environment variable set for `DATABASE_URL`, postgres_kernel will connect to a database there.
